frontend/apps/desktop/src/app/pages/page-tournament-details/header/tournament-details-header.component.less

```diff
@@ -0,0 +1,45 @@
+@import "../../../../less/colors-desktop.less";
```

Comment:
Ich kann mal gucken, dass wir colors in eine Domain packen, damit du nur noch `@import "colors" ` machen musst

Reply:
Danke, das hab ich noch nicht hinbekommen, das wäre cool

Reply:
Habe ich gerade umgesetzt: [hier](https://github.com/Nalochen/jtr-2.0/pull/90/commits/25d31b9f98adeef01be5e7aad87854fe7bce2419)